test/express-app/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "express-app",
   "description": "express-app",
-  "version": "0.0.0"
+  "version": "0.0.0",
+  "main": "server.js"
```

Comment:
I am using `require.resolve` to get an absolute path to the express app, and that does not work with `server.js`